decoder/label_candidates.go

```diff
@@ -81,6 +95,19 @@ func (d *Decoder) labelCandidatesFromDependentSchema(idx int, db map[schema.Sche
 	return candidates, nil
 }
 
+func generateSnippet(label string, attr map[string]*schema.AttributeSchema) string {
```

Comment:
```suggestion
func generateRequiredFieldsSnippet(label string, attr map[string]*schema.AttributeSchema) string {
```
We have other functions which generate snippets and functions in Go are package-wide, so I think the name here should help differentiate this function from the others.